src/main/java/seedu/address/logic/commands/DeleteJobCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.job.Job;
+
+/**
+ * Deletes a job identified using it's displayed index from the address book.
+ */
+public class DeleteJobCommand extends DeleteCommand<Job> {
+    public static final String ENTITY_WORD = "job";
+    public static final String MESSAGE_DELETE_JOB_SUCCESS = "Job %1$s - %2$s "
```

Comment:
Just curious, is there any particular reason we are using this format?

Reply:
Not really. I believe this had been present since AB3, just modified to suit our needs.